nmmo/core/action.py

```diff
@@ -48,7 +45,7 @@ def leaf():
   def N(cls, config):
     return len(cls.edges)
 
-  def deserialize(realm, entity, index, obs: Observation):
+  def deserialize(realm, entity, index: int, obs):
```

Comment:
why remove type hint?

Reply:
I put back type hint where it's helpful, including here.